### PR TITLE
CPLP-1965 add checklist worker

### PR DESCRIPTION
**Pull request by Phil91**

### CODE REVIEW

File: charts/portal/values.yaml [hunk 1]
```diff
@@ -122,6 +122,14 @@ backend:
     secret:
       name: "secret-provisioning-migrations"
       appsettings: ""
+  checklistworker:
+    name: "checklist-worker"
+    image:
+      name: "ghcr.io/catenax-ng/product-portal-backend_checklist-worker"
+      checklistworkertag: "int-v0.6.0"
```
Comment (evegufy) [inline charts/portal/values.yaml:129]: please add here also the v1.0.0-RC1 tag without quotes.

File: charts/portal/templates/secret-backend-portal-checklist.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+###############################################################
```
Comment (evegufy) [inline charts/portal/templates/secret-backend-portal-checklist.yaml:1]: could you please remove the 'portal' from the file?
It's not a perfect rule but it's reasonable to assume that there won't be a checklist worker for provisioning so it's not need it.

File: charts/portal/templates/cronjob-backend-checklist.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+kind: CronJob
+metadata:
+  name: { { .Values.backend.checklistworker.name } }
+spec:
+  schedule: "0 0 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: OnFailure
```
Comment (evegufy) [inline charts/portal/templates/cronjob-backend-checklist.yaml:30]: please add: concurrencyPolicy: Forbid

**Review by evegufy — CHANGES_REQUESTED** [review]
comments

File: charts/portal/values.yaml (hunk 1 above)
Comment (Phil91) [reply charts/portal/values.yaml]: changed

File: charts/portal/templates/secret-backend-portal-checklist.yaml (hunk 2 above)
Comment (Phil91) [reply charts/portal/templates/secret-backend-portal-checklist.yaml]: changed it to checklist-worker

File: charts/portal/templates/cronjob-backend-checklist.yaml (hunk 3 above)
Comment (Phil91) [reply charts/portal/templates/cronjob-backend-checklist.yaml:30]: added

**Comment by Phil91** [comment]
> comments

thanks for reviewing. isseus should be resolved

**Review by evegufy — CHANGES_REQUESTED** [review]

File: charts/portal/values.yaml [hunk 4]
```diff
@@ -122,6 +122,14 @@ backend:
     secret:
       name: "secret-provisioning-migrations"
       appsettings: ""
+  checklistworker:
+    name: "checklist-worker"
+    image:
+      name: "ghcr.io/catenax-ng/product-portal-backend_checklist-worker"
```
[the remark this answers is not in the document]
Comment (Phil91) [reply charts/portal/values.yaml]: changed :)

**Review by evegufy — APPROVED** [review]
lgtm, let's merge once https://github.com/catenax-ng/tx-portal-backend/pull/72 is ready

**Pull request merged**